packages/cli/test/dev/fixtures/26-nextjs-secrets/vercel.json

```diff
@@ -1,10 +1,10 @@
 {
   "build": {
     "env": {
-      "SECRET": "@buildtime"
+      "SECRET": "buildtime"
     }
   },
   "env": {
-    "ANOTHER_SECRET": "@runtime"
+    "ANOTHER_SECRET": "runtime"
```

Comment:
I'm pretty sure the `@` is necessary, otherwise its not testing secrets

Reply:
Previously this test wasn't deployed but is now required but fails since the secrets don't exist. This used to use the `@vercel/next` builder which wasn't fully testing secrets so maybe we want to rename this to `26-nextjs-env`, wdyt @TooTallNate?

Reply:
I think we should remove this test completely. The `dev-server.ts` file in vercel/next doesn't even exist anymore.

Reply:
Updated in https://github.com/vercel/vercel/pull/7822/commits/97dfc46080b76096d95ee8cef3917a5b1af0777a